### DISCUSSION TITLE
Differences of using `intellisenseMode` of `arm` vs `x86`

### DISCUSSION THREAD

**Discussion by AgainPsychoX** [microsoft/vscode-cpptools#9764] [Q&A]
What are differences of using `intellisenseMode` of `arm` vs `x86`?

**Comment by sean-mcmanus** [comment]
There are differences in how our parser runs, such as the system defines that are automatically defined, and for example a "long double" has a size of 8 instead of 16 on arm (but some differences are dependent on the OS as well).
